easybuild/easyconfigs/b/Biggus/Biggus-0.13.0-intel-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'PythonPackage'
+
+name = 'Biggus'
+version = '0.13.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/SciTools/biggus/'
+description = """Virtual large arrays and lazy evaluation."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = ['https://github.com/SciTools/Biggus/archive/']
```

Comment:
there is a source URL for Biggus 0.13.0 at https://pypi.python.org/pypi/Biggus/0.13.0

Seems to me like this *should* work... The actual download URLs are mangled by PyPI (they include a hash), but EasyBuild knows how to deal with that...

Can you try and pull the relevant part from a debug log to figure out why it's not working?